Specification.md

```diff
@@ -40,9 +40,11 @@ seq=11                                  \
 
 ## 2. Format and Naming
 
-**Sync Interest Format**: `/<group-prefix>/<state-vector>/<signature>`
+**Sync Interest Format**: `/<group-prefix>/v=2/<signature>`
 
-A state vector is appended to the name in TLV format:
+The State Vector is encoded in TLV format and included in the Application Parameters of the Sync Interest. The State Vector SHOULD be the first TLV block in the Application Parameters.
+
+Application Paramters: StateVector
```

Comment:
Like a previous comment, not sure what this is supposed to indicate, or whether it adds any useful info beyond the text above it.